src/components/Good/Good.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { GoodPropTypes } from '../PropTypes/GoodPropTypes';
+
+export const Good = ({ good, selectedGoods, onClick }) => (
+  <button
+    type="button"
+    onClick={() => onClick(good.name)}
+    className={classNames({
+      'ui download primary button': true,
+      'ui download primary button--selected': selectedGoods.includes(good.name),
```

Comment:
не працює select, такого класу наче немає в ui навіть, потрібно вручну дописувати або дивитись, які там є стилі, і плюс ти додаєш стилі два рази
```suggestion
      'ui download primary button': true,
      'button--selected': selectedGoods.includes(good.name),
```

Reply:
выделенный селект не видно